integration/test_client_storage.py

```diff
@@ -245,68 +245,78 @@ async def test_client_side_state(
     state_var_input.send_keys("c1")
     input_value_input.send_keys("c1 value")
     set_sub_state_button.click()
+    time.sleep(0.5)
     state_var_input.send_keys("c2")
     input_value_input.send_keys("c2 value")
     set_sub_state_button.click()
+    time.sleep(0.5)
     state_var_input.send_keys("c4")
     input_value_input.send_keys("c4 value")
     set_sub_state_button.click()
+    time.sleep(0.5)
     state_var_input.send_keys("c5")
     input_value_input.send_keys("c5 value")
     set_sub_state_button.click()
+    time.sleep(0.5)
```

Comment:
are all these sleeps really necessary? what changed about the test to need them?

Reply:
@masenf  This should be removed now. There seem to have been some race condition when I ran with redis locally that prompted the sleeps but they pass the CI now.